src/sentry/mediators/external_issues/issue_link_creator.py

```diff
@@ -35,23 +33,13 @@ def _make_external_request(self):
             action=self.action,
         )
 
-    def _format_response_data(self):
-        web_url = self.response["webUrl"]
-
-        display_name = "{}#{}".format(
-            escape(self.response["project"]), escape(self.response["identifier"])
-        )
-
-        return [web_url, display_name]
-
     def _create_external_issue(self):
-        web_url, display_name = self._format_response_data()
-        self.external_issue = PlatformExternalIssue.objects.create(
-            group_id=self.group.id,
-            project_id=self.group.project_id,
-            service_type=self.sentry_app.slug,
-            display_name=display_name,
-            web_url=web_url,
+        self.external_issue = external_issues.Creator.run(
+            install=self.install,
+            group=self.group,
+            web_url=self.response["webUrl"],
+            project=self.group.project.slug,
```

Comment:
so this needs to be `self.response["project"]` because it's not actually the sentry project, but rather the "project" from the external service. not great naming here on that tho :/

Reply:
ohhhh oops lol